ibm_catalog.json

```diff
@@ -237,22 +239,20 @@
                 }
               ],
               "optional": true,
-              "on_by_default": true,
-              "version": "^v2.0.6"
+              "on_by_default": false,
+              "version": "v3.0.7"
             },
             {
-              "name": "testing-deploy-arch-ibm-observability",
-              "id": "daf2cf91-2384-4205-b7f4-621299a4896d-global",
-              "version": "^1.3.0",
+              "name": "deploy-arch-ibm-observability",
+              "description": "Cloud Automation for Observability supports configuring IBM Cloud Observability resources for logging, monitoring, and activity tracker event routing.",
```

Comment:
https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-orchestrate/pull/81/files#diff-05d815f1542258c2858633435a8fa9b0419177ca18e4ce23d64eeeb0efd41384R219
Update the description to something similar to this